notebooks/official/tabular_workflows/bqml_arima_plus.ipynb

```diff
@@ -29,22 +29,22 @@
         "id": "title"
```

Comment:
<p>Update the links</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
:man_facepalming: thanks